client/core/types.go

```diff
@@ -480,27 +480,29 @@ func newDisplayIDFromSymbols(base, quote string) string {
 	return strings.ToUpper(base) + "-" + strings.ToUpper(quote)
 }
 
-// MiniOrder is minimal information about an order in a market's order book.
-type MiniOrder struct {
-	Qty   float64 `json:"qty"`
-	Rate  float64 `json:"rate"`
-	Epoch uint64  `json:"epoch,omitempty"`
-	Sell  bool    `json:"sell"`
-	Token string  `json:"token"`
+// SimpleOrder is minimal information about an order in a market's order book.
+// Replaced MiniOrder, which had a float Qty in conventional units.
+type SimpleOrder struct {
+	Qty   uint64 `json:"qty"`
+	Rate  uint64 `json:"rate"`
```

Comment:
So if I were a consumer of the orderbook rpc, which returns `SimpleOrder`s, which functions could I use to get from this `Rate` (message rate or is it atomic rate?) to conventional rate?

If it's too much of a burden for the consumer to get all the conversion factors from the assets' unitinfos, perhaps we have more than one rate field here like `Rate float64` for the conventional rate and `RateAtomic uint64` for the other?